rootdir/init.hawaii_ss_kylepro.rc

```diff
@@ -29,9 +27,6 @@ on init
     symlink /storage/emulated/legacy /mnt/sdcard
     symlink /storage/emulated/legacy /storage/sdcard0
     symlink /mnt/shell/emulated/0 /storage/emulated/legacy
-    symlink /storage/sdcard1 /extSdCard
-    symlink /storage/sdcard1 /mnt/extSdCard
-    symlink /storage/sdcard1 /external_sd
```

Comment:
those are for compatibility purposed, some stock libs expected to have extSdCard